apps/parsley/src/components/DetailsMenu/DetailsMenuCard/Toggles/HighlightFiltersToggle/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { usePreferencesAnalytics } from "analytics";
+import { useLogContext } from "context/LogContext";
+import BaseToggle from "../BaseToggle";
+
+const HighlightFiltersToggle: React.FC = () => {
+  const { sendEvent } = usePreferencesAnalytics();
+  const { preferences } = useLogContext();
+  const { highlightFilters, setHighlightFilters } = preferences;
+  const isChecked = highlightFilters;
+
+  const onChange = (checked: boolean) => {
+    sendEvent({ name: "Toggled Highlight Filters", on: checked });
+    setHighlightFilters(checked);
+  };
+  return (
+    <BaseToggle
+      data-cy="highlight-filters-toggle"
+      label="Add Highlights To Filters"
```

Comment:
[nit] Could be briefer
```suggestion
      label="Highlight Filters"
```